pkg/frost/party/id.go

```diff
@@ -12,10 +13,10 @@ import (
 // ByteSize is the number of bytes required to store and ID or Size
 const ByteSize = 2
```

Comment:
Suboptimal naming: `ByteSize` without the context sound like "the size of a byte", rename to `IDByteSize`?